src/main/java/DukeCommand.java

```diff
@@ -0,0 +1,10 @@
+public enum DukeCommand {
+    BYE("bye"), MARK("mark"), UNMARK("unmark"), LIST("list"), TODO("todo"), DEADLINE(
+            "deadline"), EVENT("event");
```

Comment:
I like how you extracted out possible commands and make it into an `Enum` class!